### PR TITLE
25353 exponential functions

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added examples to CSS exponential functions `hypot()`, `log()` and `sqrt()`

### Motivation

Working on Issue [25353](https://github.com/mdn/content/issues/25353), which is related to bu [1814469](https://bugzilla.mozilla.org/show_bug.cgi?id=1814469)

### Additional details

N/A

### Related issues and pull requests

- [Browser Compat Data](https://github.com/mdn/browser-compat-data/pull/19366)
- [Firefox Release](https://github.com/mdn/content/pull/26203)

### CODE REVIEW

File: files/en-us/web/css/hypot/index.md [hunk 1]
```diff
@@ -42,6 +40,82 @@ Returns a {{CSSxRef("&lt;number&gt;")}}, {{CSSxRef("&lt;dimension&gt;")}}, or {{
...
+</div>
+```
+
+#### CSS
+
+Here we are using [CSS custom properties](/en-US/docs/Web/CSS/Using_CSS_custom_properties) to define the sizes to be used. First we declare the first size (`--size-0`) which is then used to calculate the other sizes.
+
+- `--size-1` is calculated with the hypotenuse of `--size-0` (100px), this takes square the value and as there is no other value returns the square root of the value which results in 100px
+- `--size-2` is calculated with the hypotenuse of `--size-0` (100px) twice, this takes square the value (10000) and adds it to the square of `--size-0` again (20000) returns the square root of 20000 which results in 141.417px
+- `--size-3` is calculated with the hypotenuse `--size-0` _ 1.5 (150px) and `--size-0` _ 2 (200), these values are squared (22500 and 40000) and added together (62500) and then square rooted, which results in 250px
```
Comment (estelle) [inline files/en-us/web/css/hypot/index.md:66]:
```suggestion
- `--size-1` is calculated with the hypotenuse of `--size-0` (100px). This takes the square value and, as there is no other value, returns the square root of the value, which results in 100px.
- `--size-2` is calculated with the hypotenuse of `--size-0` (100px), twice. This takes the square of the value (100 * 100 = 10000) and adds it to the square of `--size-0` again (10000 + 10000 = 20000) and returns the square root of the sum (√20000), which results in 141.42px.
- `--size-3` is calculated with the hypotenuse `--size-0` _ 1.5 (150px) and `--size-0` _ 2 (200). The result is the square root of the sum of their squares: The values are squared (22500 and 40000) and added together (62500), with the sum sum square-rooted (√62500) being 250px.
```

File: files/en-us/web/css/hypot/index.md [hunk 2]
```diff
@@ -42,6 +40,82 @@ Returns a {{CSSxRef("&lt;number&gt;")}}, {{CSSxRef("&lt;dimension&gt;")}}, or {{
...
+
+```css
+:root {
+  --size-0: 100px;
+  --size-1: hypot(var(--size-0)); /*  100px */
+  --size-2: hypot(var(--size-0), var(--size-0)); /*  141.417px */
```
Comment (estelle) [inline files/en-us/web/css/hypot/index.md:72]:
```suggestion
  --size-2: hypot(var(--size-0), var(--size-0)); /*  141.42px */
```

File: files/en-us/web/css/hypot/index.md [hunk 3]
```diff
@@ -42,6 +40,82 @@ Returns a {{CSSxRef("&lt;number&gt;")}}, {{CSSxRef("&lt;dimension&gt;")}}, or {{
...
+
+This example shows how you can use the `hypot()` function to calculate sizes.
+
+#### HTML
+
+```html
+<div class="boxes">
+  <div class="box">100px</div>
+  <div class="box one">100px</div>
+  <div class="box two">141.417px</div>
```
Comment (estelle) [inline files/en-us/web/css/hypot/index.md:55]:
```suggestion
  <div class="box two">141.42px</div>
```

File: files/en-us/web/css/log/index.md [hunk 4]
```diff
@@ -45,6 +43,79 @@ The natural logarithm (base `e`) of `value`, when `base` is not defined.
...
+</div>
+```
+
+#### CSS
+
+Here we are using [CSS custom properties](/en-US/docs/Web/CSS/Using_CSS_custom_properties) to define the sizes to be used. First we declare the first size (`--size-0`) which is then used to calculate the other sizes.
+
+- `--size-1` is calculated by multiplying `--size-0` (50px) by the value of `log(7.389)` (2) which results in 100px
+- `--size-2` is calculated by multiplying `--size-0` (50px) by the value of `log(8, 2)` (3) which results in 150px
+- `--size-3` is calculated by multiplying `--size-0` (50px) by the value of `log(625, 5)` (4) which results in 200px
```
Comment (estelle) [inline files/en-us/web/css/log/index.md:69]:
```suggestion
Here we are using [CSS custom properties](/en-US/docs/Web/CSS/Using_CSS_custom_properties) to define the sizes to be used. First, we declare the first size (`--size-0`), which is then used to calculate the other sizes.

- `--size-1` is calculated by multiplying `--size-0` (50px) by the value of `log(7.389)` (2) which results in 100px.
- `--size-2` is calculated by multiplying `--size-0` (50px) by the value of `log(8, 2)` (3) which results in 150px.
- `--size-3` is calculated by multiplying `--size-0` (50px) by the value of `log(625, 5)` (4) which results in 200px.
```

File: files/en-us/web/css/sqrt/index.md [hunk 5]
```diff
@@ -41,6 +39,79 @@ Returns a {{cssxref("&lt;number&gt;")}} which is the square root of `x`.
...
+</div>
+```
+
+#### CSS
+
+Here we are using [CSS custom properties](/en-US/docs/Web/CSS/Using_CSS_custom_properties) to define the sizes to be used. First we declare the first size (`--size-0`) which is then used to calculate the other sizes.
+
+- `--size-1` is calculated by multiplying the value of `--size-0` (50px) by the square root of 4 (2) which results in 100px
+- `--size-2` is calculated by multiplying the value of `--size-0` (50px) by the square root of 9 (3) which results in 150px
+- `--size-3` is calculated by multiplying the value of `--size-0` (50px) by the square root of 16 (4) which results in 200px
```
Comment (estelle) [inline files/en-us/web/css/sqrt/index.md:65]:
```suggestion
Here we are using [CSS custom properties](/en-US/docs/Web/CSS/Using_CSS_custom_properties) to define the sizes to be used. First, we declare the first size (`--size-0`), which is then used to calculate the other sizes.

- `--size-1` is calculated by multiplying the value of `--size-0` (50px) by the square root of 4 (2), which results in 100px.
- `--size-2` is calculated by multiplying the value of `--size-0` (50px) by the square root of 9 (3), which results in 150px.
- `--size-3` is calculated by multiplying the value of `--size-0` (50px) by the square root of 16 (4), which results in 200px.
```

File: files/en-us/web/css/sqrt/index.md [hunk 6]
```diff
@@ -41,6 +39,79 @@ Returns a {{cssxref("&lt;number&gt;")}} which is the square root of `x`.
 
 {{CSSSyntax}}
 
+## Examples
+
+### Scale sizes based on square root
+
+This example shows how you can use the `sqrt()` function to calculate sizes.
```
Comment (estelle) [inline files/en-us/web/css/sqrt/index.md:46]: A sentence as to why this would be useful would be helpful (not required, but helpful) ;)

**Review by estelle — CHANGES_REQUESTED** [review]
A few grammar tweaks, one suggestion, and one edit. Looks good. 

It would be helpful (though fine as is with the suggested edits confirmed) to add something about when these functions would be useful in the example, like maybe adding a 1px diagonal line using generated content that goes from corner to corner using Pythagorean theory - but something actually useful ;). I can't come up with anything.

File: files/en-us/web/css/sqrt/index.md (hunk 6 above)
Comment (dletorey) [reply files/en-us/web/css/sqrt/index.md:46]: I have spent way too long reading about Square roots and trying to find an use case for this and failing badly

**Review by Rumyra — APPROVED** [review]

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**